oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -151,6 +155,46 @@ public AccessToken refreshAccessToken() throws IOException {
     return new AccessToken(accessToken, new Date(expiresAtMilliseconds));
   }
 
+  /**
+   * Returns an Google ID Token from the metadata server on ComputeEngine
+   * 
+   * @param targetAudience the aud: field the IdToken should include
+   * @param options list of Credential specific options for for the
+   * token. For example, an IDToken for a ComputeEngineCredential could have 
```

Comment:
continuation lines should be indented four spaces past the asterisk. google-java-format plugins for Eclipse and IntelliJ should do this automatically

Reply:
i'm using vscode. is i dind't find a plugin formatter for that.
the spaces in this example start off at the text possiton for the description of the param
https://github.com/googleapis/google-auth-library-java/blob/master/oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java#L222

if the guide is just 4 spaces post `*`, thats fine, i'll alter it but i'm looking at the existing sample as reference

Reply:
We enabled a maven plugin to assist with formatting. Please merge from master and run `mvn com.coveo:fmt-maven-plugin:format`

Reply:
ok, i ran that and added in some testcases for all the credential types.